concepts/observability-strategy/opentelemetry-operator/README.md

```diff
@@ -0,0 +1,89 @@
+- The user can specify custom `OpenTelemetryCollector` resource for custom exporter handling or any other activities likes running a sidecar.
+- The user can specify custom `Instrumentation` for env injection of custom collectors.
+
+![Architecture](./assets/architecture.drawio.svg)
+
+## POC
+
+### Prerequisites
+- Have a Kyma cluster in version >= 2.10
+- Have a TracePipeline configured with a valid output
+
+
+### Installation
+Install the Otel Operator using helm
+
+```bash
+helm upgrade otel-operator open-telemetry/opentelemetry-operator --version 0.21.1 --install --namespace kyma-system -f otel-operator-values.yaml
+```
+
+Create an Instrumentation configuration with the endpoints to use
+```bash
+kubectl apply -n kyma-system -f instrumentation.yaml
+```
+
+### Instrumentation
+Annotate your app with either:
+```yaml
+annotations:
+  instrumentation.opentelemetry.io/inject-sdk: "kyma-system/kyma" #to inject env variables only
+  instrumentation.opentelemetry.io/inject-nodejs: "kyma-system/kyma" #to inject env variables and nodejs instrumentation library
+  instrumentation.opentelemetry.io/container-names: "myapp,myapp2" #to inject env variables and javaagent
+```
+
+### Findings
+
+The injection of environment variables does not only inject the destination for the exporters but also injects the typical OTEL resource attributes. Also it supports injecting trace propagation and sampling strategies. That is a very convenient feature to configure your workload declarative in case of it is already instrumented with the OTEL SDK.
+
+The auto-instrumentation was tested for java and nodejs. Here, we failed to get the nodejs instrumentation working at all, one application was coming up operational but without instrumentation active. Another one was even not starting up anymore.
+
+## Considerations, Risks, Potentials
+
+In the following several very relevant aspects are getting discussed in order to derive a architectural decision
+
+### Simplification of own abstraction logic
+
+The OpenTelemetry Operator is a well maintained project having an active community behind. The operator maps an otel-collector configuration in an actual deployment and manages the lifecycle of it. With that, the telemetry-operator does not need to manage concrete deployments/daemonsets and alike anymore but could rely on the intermediate abstraction and with that could get a simplified implementation. New features in the underlying operator can be leveraged immediately in the layer above.
+
+However, the current functionality of the OpenTelemetry Operator in regards to deployments is very limited. It mainly maps the collector definition 1:1 to the underlying k8s resource. Actual logic which will simplify more complex scenarios are not covered, also simple scenarios like adding a batch processor in combination with a memory-ballast extension with settings fitting to the actual resource settings are not covered. So there is simplification in pure resource handling mainly.
```

Comment:
logic either singular or plural. I'm guessing singular here; if I'm wrong, write "actual logics ... are not covered" for plural.

```suggestion
However, the current functionality of the OpenTelemetry Operator in regards to deployments is very limited. It mainly maps the collector definition 1:1 to the underlying Kubernetes resource. Actual logic, which will simplify more complex scenarios, is not covered. Also simple scenarios are not covered, like adding a batch processor in combination with a memory-ballast extension with settings fitting to the actual resource settings. So there is simplification in pure resource handling mainly.
```